Ginger/Ginger/MainWindow.xaml.cs

```diff
@@ -120,11 +120,6 @@ public void Init()
                 if (WorkSpace.Instance.UserProfile.GingerStatus == eGingerStatus.Active)
                 {
                     Reporter.ToStatus(eStatusMsgKey.ExitMode);
-                    WorkSpace.Instance.UserProfile.DoNotAskToRecoverSolutions = false;
```

Comment:
This part detects the if application was crashed previously. do not remove it.
Only remove recover label and check box from edit user settings page